Emby.Dlna/ContentDirectory/ControlHandler.cs

```diff
@@ -908,6 +908,16 @@ private QueryResult<ServerItem> GetMovieFolders(BaseItem item, User user, StubTy
                 }
             };
 
+            // If Limit is < 6 then we return too many items, so we must remove some.
+            if (limit != null && limit < array.Length)
```

Comment:
```suggestion
            if (limit < array.Length)
```

Reply:
Well you learn something new every day!

Didn't know that the comparisons  > < <= and => with a null always return false. Will change code.